src/policy/fees.cpp

```diff
@@ -390,8 +390,9 @@ void CBlockPolicyEstimator::processTransaction(const CTxMemPoolEntry& entry, boo
         mapMemPoolTxs[hash].bucketIndex = feeStats.NewTx(txHeight, (double)feeRate.GetFeePerK());
     }
     else {
-        LogPrint("estimatefee", "not adding\n");
+        LogPrint("estimatefee", "not adding");
     }
+    LogPrint("estimatefee", "\n");
```

Comment:
Original code:
```
LogPrint("estimatefee", "Blockpolicy mempool tx %s ", hash.ToString().substr(0,10));
 // Record this as a priority estimate
if (entry.GetFee() == 0 || isPriDataPoint(feeRate, curPri)) {
...
}
// Record this as a fee estimate
else if (isFeeDataPoint(feeRate, curPri)) {
...
}
else {
    LogPrint("estimatefee", "not adding\n");
}
```
So the line will not be terminated unless the if() condition falls through.

But if you insist I'll remove this change...